main.go

```diff
@@ -32,6 +32,7 @@ import (
 	operatorv3 "github.com/IBM/ibm-common-service-operator/api/v3"
 	"github.com/IBM/ibm-common-service-operator/controllers"
 	"github.com/IBM/ibm-common-service-operator/controllers/bootstrap"
+	certmanager "github.com/IBM/ibm-common-service-operator/controllers/certManager"
```

Comment:
folder name should match the package name, so should be `controllers/certmanager/`

Reply:
I refer it from here https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/render_template.go#L28

But I will update the folder name to `controllers/certmanager` to keep consistency.

Reply:
yea, I need to change that. It doesn't seem to be following conventions in the rest of the project